### PR TITLE
Fix: missed pagination from the page of transactions, referring cells and deployed cells. 

### PR DESCRIPTION
**Pull request by Daryl-L**
reference: https://github.com/Magickbase/ckb-explorer-public-issues/issues/440

### CODE REVIEW

**Review by Keith-CY — APPROVED** [review]

**Comment by FrederLu** [comment]
There are 2 styling issues with the UI on mobile:
1. When paging, there is no need to display a white bottom frame below the operation bar;
2. The right margin of the "GoTo" button is too small, please refer to UI adjustment;

image
image

File: src/pages/Script/ScriptsComp.tsx [hunk 1]
```diff
@@ -39,14 +39,14 @@ export const ScriptTransactions = ({ page, size }: { page: number; size: number
         },
       })
       .then((res: AxiosResponse) =>
-        toCamelcase<Response.Response<{ ckbTransactions: CkbTransactionInScript[] }>>(res.data),
+        toCamelcase<Response.Response<{ ckbTransactions: CkbTransactionInScript[]; meta: PageInfo }>>(res.data),
```
Comment (WhiteMinds) [inline src/pages/Script/ScriptsComp.tsx:42]: Strange, why is the backend API using this response format?

Based on the type, shouldn't `meta` have been provided at the root level before?

Comment (Keith-CY) [reply src/pages/Script/ScriptsComp.tsx:42]: cc @rabbitz

Comment (Daryl-L) [reply src/pages/Script/ScriptsComp.tsx:42]: > Strange, why is the backend API using this response format?
> 
> Based on the type, shouldn't `meta` have been provided at the root level before?

`data` maybe the root level of the response structure, with other properties like `message` or `status` always for restful API.

Comment (rabbitz) [reply src/pages/Script/ScriptsComp.tsx:42]: https://github.com/nervosnetwork/ckb-explorer/commit/04b8ae9ff618254fd8c566afb9b21f19d1fd1094#diff-09a97666b995d34f0472095b2c99024773a75e4640cba18e3510085e5ed315f2R27

This return data structure was defined by the previous author and has been deployed for the interface. I assumed it was an agreed-upon structure with the frontend.

Comment (WhiteMinds) [reply src/pages/Script/ScriptsComp.tsx:42]: At the time, we discussed the response structure convention during a meeting and in https://github.com/Magickbase/ckb-explorer-public-issues/issues/315#issuecomment-1541319430. At that time, it was agreed that meta would be directly under the root level.

Comment (Daryl-L) [reply src/pages/Script/ScriptsComp.tsx:42]: > At the time, we discussed the response structure convention during a meeting and in [Magickbase/ckb-explorer-public-issues#315 (comment)](https://github.com/Magickbase/ckb-explorer-public-issues/issues/315#issuecomment-1541319430). At that time, it was agreed that meta would be directly under the root level.

If so, the API should be adjust.

**Comment by Daryl-L** [comment]
> There are 2 styling issues with the UI on mobile:
> 
> 1. When paging, there is no need to display a white bottom frame below the operation bar;
> 2. The right margin of the "GoTo" button is too small, please refer to UI adjustment;
> 
> image image

done

**Comment by FrederLu** [comment]
> > There are 2 styling issues with the UI on mobile:
> > 
> > 1. When paging, there is no need to display a white bottom frame below the operation bar;
> > 2. The right margin of the "GoTo" button is too small, please refer to UI adjustment;
> > 
> > image image
> 
> done

https://ckb-explorer-frontend-in-magickbase-repo-k2glzivhd-magickbase.vercel.app/script/0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8/type
@Daryl-L 
Problem 1 still exists. Issue 2 has been fixed.
Please confirm.

image

**Comment by Daryl-L** [comment]
> > > There are 2 styling issues with the UI on mobile:
> > > 
> > > 1. When paging, there is no need to display a white bottom frame below the operation bar;
> > > 2. The right margin of the "GoTo" button is too small, please refer to UI adjustment;
> > > 
> > > image image
> > 
> > 
> > done
> 
> https://ckb-explorer-frontend-in-magickbase-repo-k2glzivhd-magickbase.vercel.app/script/0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8/type @Daryl-L Problem 1 still exists. Issue 2 has been fixed. Please confirm.
> 
> image

I missed it and it was done just now in https://github.com/Magickbase/ckb-explorer-frontend/pull/113/commits/994f730137120c43aa64d89ad031f5fd3ac78b81

**Comment by FrederLu** [comment]
>I missed it and it was done just now in https://github.com/Magickbase/ckb-explorer-frontend/commit/994f730137120c43aa64d89ad031f5fd3ac78b81

This modification was successful and the bottom is now normal.
But I found that `Code Hash` and `Type Id` are not displayed correctly. Need to confirm whether this modification caused the error.
image

image

**Comment by Daryl-L** [comment]
> > I missed it and it was done just now in https://github.com/Magickbase/ckb-explorer-frontend/commit/994f730137120c43aa64d89ad031f5fd3ac78b81
> 
> This modification was successful and the bottom is now normal. image But I found that `Code Hash` and `Type Id` were displayed incorrectly, and `Referring Cells (4,267,580)` was partially obscured. Need to confirm whether this modification caused the error. https://ckb-explorer-frontend-in-magickbase-repo-7ymtfcixc-magickbase.vercel.app/script/0x9bd7e06f3ecf4be0f2fcd2188b23f1b9fcc88e5d4b65a8637b17723bbda3cce8/type?page=1&size=10 image

done

**Review by FrederLu — APPROVED** [review]

File: src/pages/Script/ScriptsComp.tsx (hunk 1 above)
Comment (WhiteMinds) [reply src/pages/Script/ScriptsComp.tsx:41]: For this issue, another issue needs to be created to be handled by the backend.
https://github.com/Magickbase/ckb-explorer-public-issues/issues/451

**Pull request merged**